.neutrinorc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         .rule('js-yaml')
           .test(/\.(yaml|yml)$/)
           .use('js-yaml-loader')
+          .options({safe: true})
```

Comment:
Can you point me where I can find more information about this option?

Reply:
Clicking on the link shows:
![screen shot 2018-10-04 at 11 28 56 am](https://user-images.githubusercontent.com/3766511/46484916-b9d60e00-c7c8-11e8-9491-08e99653f281.png)

Reply:
@helfi92 ah sorry! I tried pointing it to the README file but it ends up not to be a link x_x here's the link https://github.com/wwilsman/js-yaml-loader/compare/v0.1.0...v1.0.0 then go to the readme file part

Reply:
Thank you!